scripts/new_project.kts

```diff
@@ -213,6 +222,18 @@ object NewProject {
         }
     }
 
+    private fun validateDestination(value: String) {
+        if (value.isNotBlank()) {
+            destination = "${value.trim()}$SEPARATOR_SLASH"
+        } else {
+            showMessage(
+                message = "Error: Invalid Destination: destination cannot be blank \n$helpMessage",
+                exitAfterMessage = true,
+                isError = true,
+            )
+        }
+    }
```

Comment:
AC says
> If no argument is given, default to the template root folder

So we need to change the logic here not to show an error message if the `destination` is blank.

Reply:
@kaungkhantsoe Just to confirm, if we use these commands, it'll default to the current directory.
```
kscript new_project.kts package-name=co.nimblehq.suvkmmsurveys.android app-name=Surveys template=compose destination=
// or
kscript new_project.kts package-name=co.nimblehq.suvkmmsurveys.android app-name=Surveys template=compose destination=""
```
Note the `destination` argument with nothing specified after the equal sign or a blank string.

Reply:
Got it 🚀